tests/Microsoft.Bot.Builder.TemplateManager/TemplateManagerTests.cs

```diff
@@ -1,120 +1,90 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using System.Threading.Tasks;
 using Microsoft.Bot.Builder.Adapters;
 using Microsoft.Bot.Schema;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Xunit;
 
 namespace Microsoft.Bot.Builder.TemplateManager.Tests
 {
-    [TestClass]
-    [TestCategory("Template")]
-    public class TemplateManagerTests
+    [Trait("TestCategory", "Template")]
+    public class TemplateManagerTests : IClassFixture<TemplateFixture>
     {
-        private static TestContext _testContext;
-        private static LanguageTemplateDictionary templates1;
-        private static LanguageTemplateDictionary templates2;
+        private readonly TemplateFixture templateFixture;
```

Comment:
Private member should start with `_`
```suggestion
        private readonly TemplateFixture _templateFixture;
```